dev/run_shard.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# import utils functions
+dev_dir="$(dirname "$0")"
+. $dev_dir/utils/utils.sh
+
+function run_shard(){
+  local minikube_ip=$1
+  local webhook_tech_token=$2
+
+  mvn clean compile \
+    -f $dev_dir/../shard-operator/pom.xml \
+    -Dquarkus.http.port=1337 \
+    -Dminikubeip=$minikube_ip \
+    -Devent-bridge.webhook.technical-bearer-token=$webhook_tech_token \
+    -Pminikube \
+    quarkus:dev
+}
+
+function retrieve_webhook_tech_token(){
+  local minikube_ip=$1
+  KEYCLOAK_RESPONSE=$(curl --insecure -X POST http://$minikube_ip:30007/auth/realms/event-bridge-fm/protocol/openid-connect/token --user event-bridge:secret -H 'content-type: application/x-www-form-urlencoded' -d 'username=webhook-robot-1&password=therobot&grant_type=password&scope=offline_access')
```

Comment:
Print the curl response here, it might be useful

Reply:
Printed in case of failures, thanks